aiohttp/helpers.py

```diff
@@ -748,7 +748,11 @@ def _parse_content_type(self, raw: str) -> None:
             self._content_type = "application/octet-stream"
             self._content_dict = {}
         else:
-            self._content_type, self._content_dict = cgi.parse_header(raw)
+            msg = HeaderParser().parsestr("Content-Type: " + raw)
+            self._content_type = msg.get_content_type()
+            params = msg.get_params()
+            del params[0]  # content type duplicated here
+            self._content_dict = dict(params)
```

Comment:
```suggestion
            self._content_dict = dict(params[1:])  # First element is content type again
```
Maybe a list slice is simpler?

Reply:
Yes, definitely, thanks!